crypto/s2n_sequence.c

```diff
@@ -38,3 +42,17 @@ int s2n_increment_sequence_number(struct s2n_blob *sequence_number)
 
     return 0;
 }
+
+int s2n_sequence_number_to_uint64(struct s2n_blob *sequence_number, uint64_t *output)
+{
+    notnull_check(sequence_number);
```

Comment:
I wonder if we should be a little more defensive here, and have an actual type called `s2n_sequence_number`.  It could have an underlying blob, but would prevent cases where we accidentally pass things that are not sequences numbers into these functions.

Reply:
Not necessarily a fix for this PR, but you are already in this code, so it might be easy.

Reply:
I feel like that would touch a lot of code, so I'm going to leave it out of this pr, but we should fix this.